src/color_range.cpp

```diff
@@ -32,7 +32,7 @@
 namespace
 {
 #ifdef HAVE_CXX20
-std::vector<color_t> recolor_range_impl(const color_range& new_range, std::span<color_t> old_rgb)
+std::vector<color_t> recolor_range_impl(const color_range& new_range, std::span<const color_t> old_rgb)
```

Comment:
I don't exactly have an issue with merging this, but it doesn't fit the description of the PR – "Various changes from https://github.com/wesnoth/wesnoth/pull/9109 that don't need C++20." This one clearly requires C++20.

Reply:
True. I figured it was worth just leaving in, though, given it was behind the include guard.